Source/WebCore/xml/XMLHttpRequest.cpp

```diff
@@ -524,13 +523,13 @@ ExceptionOr<void> XMLHttpRequest::send(Blob& body)
         if (!m_url.url().protocolIsInHTTPFamily()) {
             // FIXME: We would like to support posting Blobs to non-http URLs (e.g. custom URL schemes)
             // but because of the architecture of blob-handling that will require a fair amount of work.
```

Comment:
I wasn't sure about this one. Unless there is some extension way to go beyond https://fetch.spec.whatwg.org/#fetch-scheme it seems to me that non-HTTP(S) doesn't work but also shouldn't work.

Reply:
We should probably keep it.
I am not sure how much fetch can actually describe custom schemes.